package.json

```diff
@@ -36,26 +36,31 @@
     "eslint": "eslint .",
     "test": "mocha",
     "jsdoc": "jsdoc lib/* -d dist/doc",
-    "travis": "npm run lint && npm run test"
+    "travis": "npm run lint && npm run test",
+    "browserify": "mkdirp dist && node_modules/browserify/bin/cmd.js lib/index.js -o dist/pagexray.js --standalone PageXray -p [ browserify-banner --template 'PageXray v<%= pkg.version %> - Xray your HAR file and know all about the page - https://github.com/sitespeedio/pagexray' ]",
```

Comment:
I'd prefer renaming the browserify script, since it collides with the executable from the browserify module. Then "node_modules/browserify/bin/cmd.js" could be replaced with just "browserify".